plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/ConclaveTask.kt

```diff
@@ -94,9 +94,9 @@ abstract class ConclaveTask : DefaultTask() {
         }
     }
 
-    fun commandWithOutput(vararg command: Any): String {
+    fun commandWithOutput(vararg command: String, workingDir: String? = null): String {
```

Comment:
I don't think the varags works well with a parameter after it.

Reply:
I think it that it works if you specify the names of the parameters (when you use both)